tests/sitemaps_tests/test_generic.py

```diff
@@ -19,3 +30,23 @@ def test_generic_sitemap(self):
 </urlset>
 """ % expected
         self.assertXMLEqual(response.content.decode(), expected_content)
+
+    def test_generic_sitemap_queryset(self):
+        """Generic sitemap properly handle queryset"""
+        generic_sitemap_queryset = self.generic_sitemap.queryset.values_list(
+            "id", flat=True
```

Comment:
Use single quotes unless the string contains a single quote. Also, this could be combined with the previous line -- we allow up to 119 characters when it helps readability.